glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
```

Comment:
Typically contributions are made under the same common license as the project.  I wouldn't be inclined to accept mixed-licensing contributions.